app.config.ts

```diff
@@ -5,6 +5,9 @@ export default ({config}: ConfigContext): Partial<ExpoConfig> => {
   // we keep in the environment and out of code. For cloud builds, the secrets
   // are stored in Expo and supplied via process.env. For local builds, the
   // secrets can be stored in a .env file and loaded via direnv.
+
+  // we're overwriting fields that were previously defined in app.json, so we know they're non-null:
+  /* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
What null checks are we missing by the way? I thought the `!` annotation in the lines below was doing the assertion for us?

Reply:
My read is that the ESLint warning here is for the use of `!`, because in an ideal world you shouldn't need it (either your types should guarantee that the thing isn't null, or you should have a conditional that's checking that it's not null).

But this is a weird case, given that the structure of the config is determined by `app.json`, so I suppressed the error.

Reply:
OK, that makes sense - thanks for explaining